packages/yoga/src/BottomSheet/web/BottomSheet.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import styled from 'styled-components';
+import { bool, node } from 'prop-types';
+import Dialog from '../../Dialog';
+import { StyledDialog } from '../../Dialog/web/Dialog';
+
+const StyledBottomSheet = styled(Dialog)`
+  ${({
+    theme: {
+      yoga: {
+        components: { bottomsheet },
+      },
+    },
+  }) => `
+    align-items: end;
+    z-index: 10;
```

Comment:
I don't think it needs `z-index`, StyledBottomSheet works without `z-index`, doesn't it?

Reply:
It's not necessary, I needed to use this because Yoga's tabBar has a z-index and makes it overlay the bottom sheet.

Do you know another alternative to solve this?

<img width="183" alt="Screen Shot 2021-11-16 at 08 51 44" src="https://user-images.githubusercontent.com/27781419/141980508-72ce89ec-ce53-41b9-8968-845e57114dd8.png">

Reply:
Hey @mavericardo! I kindly suggest you then to remove it anyway and in a separate PR we remove this `z-index` for Yoga's doc which as I tested locally isn't needed. This way both our `Dialog` and `BottomSheet` we'll appear just fine there!